org/templates/basic/org/share_site_tile.html

```diff
@@ -16,7 +11,7 @@
             </a>
         </div>
         <div class="twitter">
-            <a href="http://www.twitter.com/share?url={{ request.get_host|iriencode }}"
+            <a href="http://www.twitter.com/share?url={{ request.get_host|iriencode }}&text=Every%20day%20we%20wait%2C%20more%20people%20die.%20Sign%20the%20petition%20%26%20tell%20world%20leaders%3A%20%23EndEbola%20now!"
```

Comment:
I have tested it, it comes out as expected. Is it fine to hardcore it like that?